openstf/model/fallback.py

```diff
@@ -0,0 +1,68 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from pathlib import Path
+from datetime import datetime, timedelta, timezone
+
+import pandas as pd
+import structlog
+
+
+def generate_fallback(
+    forecast_input: pd.DataFrame,
+    load: pd.DataFrame,
+    fallback_strategy: str = "extreme_day",
+) -> pd.DataFrame:
+    """Make a fall back forecast,
+    Set the value of the forecast 'quality' column to 'substituted'
+
+    Currently only fallback_strategy=extreme day is implemented which return historic profile of most extreme day.
+
+    Args:
+        forecast_input (pandas.DataFrame): dataframe desired for the forecast
+        load (pandas.DataFrame): index=datetime, columns=['load']
+        fallback_strategy (str): strategy to determine fallback. options:
+            - extreme_day: use daily profile of most extreme day
+    Returns:
+        pandas.DataFrame: Fallback forecast DataFrame with columns:
```

Comment:
This is an ongoing thing, its now easy to tell what a forecast actually is. Food for thoughts, some kind of schema or model would be really helpful, but out of scope here I guess.